### PR TITLE
fix(lnb-menu-item): add new icon spec

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

image

### CODE REVIEW

File: apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue [hunk 1]
```diff
@@ -58,7 +62,8 @@ const isSelectedMenu = (selectedMenuRoute: Location): boolean => {
         />
         <div class="text-wrapper">
             <p-i v-if="item.icon"
-                 :name="item.icon"
+                 :name="getIconName(item.icon)"
+                 :color="item.icon.color"
```
Comment (WANZARGEN) [inline apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue:66]: Question: Wouldn't there be an error if `item.icon` is of string type?

**Review by WANZARGEN — APPROVED** [review]
LGTM! I added one question!

File: apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue [hunk 2]
```diff
@@ -42,6 +42,10 @@ const isSelectedMenu = (selectedMenuRoute: Location): boolean => {
     if (!resolvedHref.endsWith('/')) resolvedHref += '/';
     return currentPath.startsWith(resolvedHref);
 };
+const getIconName = (icon): string => {
```
Comment (yuda110) [inline apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue]: nitpick: :)

```typescript
const getIconName = (icon: LNBItem['icon']): string => {
```

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue (hunk 2 above)
Comment (piggggggggy) [reply apps/web/src/common/modules/navigations/lnb/modules/LNBRouterMenuItem.vue]: But.. `LNBItem['icon']` can be `undefined`..

**Pull request merged**